### PR TITLE
Updated admin records

### PR DESCRIPTION
**Pull request by stepps00**
WIll fix https://github.com/whosonfirst-data/whosonfirst-data/issues/1334.

Related source issue: https://github.com/whosonfirst/whosonfirst-sources/issues/151

To do:
- [x] Update region placetype
- [x] Update county placetype
- [x] Update localadmin placetype
- [x] Update locality placetype
- [x] Update neighbourhood placetype
- [x] Flag localadmin and locality records as coterminous
- [x] Verify `mz:*` property values are appropriate for each placetype
  - Specifically, make sure the `mz:hierarchy_label` properties are flagged appropriately for localadmin records
- [x] ~Reset centroids (lbl, nav) for locality placetype if the centroid falls outside of the new polygons bounds.~ (file follow-up)
- [x] ~Review exclaves of Germany and Italy, and Lichtenstein geometries~ (file follow-up)
- [x] Verify source data and source name
- [x] Add wiki names
- [x] Validate geometries and records
- [x] PIP records

### CODE REVIEW

**Comment by stepps00** [comment]
Commit Notes

e93274c: update geoms and props in region records - 
  - Updates geometries and properties in existing region placetype records
c364f72: update county records, rd1 - 
  - Updates county records that matched on name and location
066aefe: update county records, rd2 - 
  - Updates county records that matched, but had slight name/location variation between WOF/source
c7c94e5: update county records, rd3 - 
  - Updates additional county records that matched, but had name/location variation between WOF/source
0962f4e: update county records, rd4 - 
  - Incorrectly updated the remainder non-matched county records
d82d57e: Revert "update county records, rd4" - 
  - Reverts commit 0962f4e

**Comment by missinglink** [comment]
Amazing work, I just had a quick glance over the diff and noticed entries like this:

```javascript
"name:eng_x_preferred":[
        "Le Chenit"
    ],
```

Is that correct per-se, to set French names as `eng`?

**Comment by stepps00** [comment]
Good catch -- these name properties will be overwritten in an upcoming commit. The wiki name import still needs to be run on the `-admin-ch` repo, so as part of that, I'll be updating the eng/deu/ita/fra name property values.

**Comment by stepps00** [comment]
This PR is ready for review. Once approved, I'll run the changes for new wiki names, validate all GeoJSON and geometries, and PIP.

**Comment by nvkelso** [comment]
Looking really nice so far!

Please do another round of dedup between "demoted" `locality` to `neighbourhood` points comparing them also against `localadmin`. Here's an example south of Geneve that should be deprecated (**Lancy**).

image

File: data/101/753/857/101753857.geojson [hunk 1]
```diff
@@ -133,7 +144,7 @@
     "wof:lang":[
         "fre"
     ],
-    "wof:lastmodified":1527641912,
+    "wof:lastmodified":1565139059,
     "wof:name":"Bernex",
     "wof:parent_id":404328823,
     "wof:placetype":"locality",
```
Comment (nvkelso) [inline data/101/753/857/101753857.geojson]: Seems like it should have other locality inside the `localadmin` it's coterminous with, like https://fr.wikipedia.org/wiki/S%C3%A9zenove?

File: data/101/766/261/101766261.geojson [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
+    "geom:bbox":"7.527453,47.4605451,7.5655475,47.4894943",
+    "geom:latitude":47.474461,
+    "geom:longitude":7.545931,
     "gn:population":4980,
     "iso:country":"CH",
-    "lbl:latitude":47.481946,
-    "lbl:longitude":7.547149,
-    "mps:latitude":47.481946,
-    "mps:longitude":7.547149,
+    "lbl:latitude":47.474461,
```
Comment (nvkelso) [inline data/101/766/261/101766261.geojson]: The old label centroid is better, is this a trend?

Comment (nvkelso) [reply data/101/766/261/101766261.geojson]: The `locality` label point should be on the settlement (the old quattroshapes + mapshaper point), while the coterminous `localadmin` label point should be on the (new) mapshaper location.

File: data/101/753/857/101753857.geojson (hunk 1 above)
Comment (stepps00) [reply data/101/753/857/101753857.geojson]: Hmm.. I do see data/404/328/367/404328367.geojson and data/101/753/857/101753857.geojson are flagged as coterminous in the branch diff.. maybe this isn't the full diff?

Comment (nvkelso) [reply data/101/753/857/101753857.geojson]: Agree it's resolved now, must have been Github funk. The other place is now correctly a neighbourhood.

File: data/101/766/261/101766261.geojson (hunk 2 above)
Comment (stepps00) [reply data/101/766/261/101766261.geojson]: This was completed in c831f9f

**Comment by stepps00** [comment]
I'll run the wiki names script against this branch shortly, validate, take care of a few minor superseding issues, then merge.

**Comment by nvkelso** [comment]
We had another heads up review today. Almost everything is resolved except some neighbourhood names with ` / ` separators which needs to be cleaned up. Then time to rock and roll.

**Pull request merged**